### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by mutugi-c**
In this milestone,
- I designed the webpage to a desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,68 +18,81 @@
...
+            ><img class="page--logo" src="images/Logo.png" alt="Website logo"
           /></a>
           <a href="#"
-            ><img class="page--menu" src="images/menu.png" alt=""
+            ><img class="page--menu" src="images/menu.png" alt="The menu icon"
           /></a>
+          <ul class="nav-list">
+            <li class="nav-list-item"><a href="#">Portfolio</a></li>
+            <li class="nav-list-item"><a href="#">About</a></li>
+            <li class="nav-list-item"><a href="#">Contact</a></li>
```
Comment (mahbubul14) [inline index.html:29]: - [ ] Kindly try to match the text color to the Figma design. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
 |

File: index.html [hunk 2]
```diff
@@ -18,68 +18,81 @@
...
           <img
             class="card--image"
             src="images/project1.png"
             alt="Tonic project image"
           />
+          <img
+            class="card--image-desktop"
+            src="images/project1_desktop.png"
+            alt="Tonic project image"
+          />
```
Comment (mahbubul14) [inline index.html:95]: - [ ] The projects should match the Figma design. Kindly work on the text color and align-items. You can check the following screenshots. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
 |

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
**Hi @mutugi-c,**

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by mutugi-c** [comment]
Hello, @mahbubul14. Thank you for pointing out these issues. I have already worked on them 😎.

**Review by amon-cofie — APPROVED** [review]
Hi @mutugi-c , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 And great job implementing all the changes the reviewer suggested 👍 



## Highlights

✔️ No linter error.
✔️ Proper Github flow used.
✔️ Good commit messages

## Optional suggestions

[OPTIONAL] Indeed you have mentioned what you did in this PR but I think that you should always make an effort to make your pull request summaries as detailed as possible. This doesn't only help your reviewer to gain insight into what you did in the pull request but also yourself as a developer when you have to refer back to this project some time in the future 😉

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**